leonardnjura/github-users
# #156037248 Revamp ui

#### What does this PR do?
- Changes list view to grid view
- Refactors ui with collapsing toolbar
- Swipe action refreshes view

#### Description of Task to be completed?
- [x] Create Toolbar
- [x] Create AppBarLayout
- [x] Create CollapsingToolbarLayout
- [x] Create RecyclerView with GridLayoutManager

#### How should this be manually tested?
* Clone this repo: 
`$ git clone -b ft-ui-revamp-156037248 https://github.com/leonardnjura/github-users`
* Open cloned directory `github-users` in Android Studio
* Build with Gradle
* Run app in emulator device or actual connected device i.e. a physical Android phone or tablet
* From the `MainActivity` take note of grid view. From the toolbar menu of `MainActivity` tap refresh option or swipe view to refresh. Then tap any user item card to view its detail components.
* From `DetailView` swipe up and down to see the collapsing toolbar with user avatar.


#### What are the relevant pivotal tracker stories?
[#156037248](https://www.pivotaltracker.com/story/show/156037248)

#### Screenshots
![ezgif com-video-to-gif (2)](https://user-images.githubusercontent.com/39657549/54753652-9ece4d80-4bf2-11e9-80c3-6e23291cf81d.gif)
